ostelco-ios-client/ThemeManager.swift

```diff
@@ -1,73 +0,0 @@
-//
-//  ThemeManager.swift
-//  ostelco-ios-client
-//
-//  Created by mac on 11/6/18.
-//  Copyright © 2018 mac. All rights reserved.
-//
-// Heavly insipred from https://medium.com/@abhimuralidharan/maintaining-a-colour-theme-manager-on-ios-swift-178b8a6a92
-
-import UIKit
-import Foundation
-
-enum Theme: Int {
-    case BlueTheme, TurquoiseTheme
-    
-    var mainColor: UIColor {
-        switch self {
-        case .BlueTheme:
-            return UIColor(named: "PeacockBlue")!
-        case .TurquoiseTheme:
-            return UIColor(named: "TurquoiseBlue")!
-        }
-    }
-    
-    var textOnMainColor: UIColor {
-        return UIColor(named: "White")!
-    }
-    
-    var logo: UIImage {
-        switch self {
-        case .BlueTheme:
-            return #imageLiteral(resourceName: "StoryboardLaunchScreenProduction")
-        case .TurquoiseTheme:
-            return #imageLiteral(resourceName: "StoryboardLaunchScreenDevelopment")
-        }
-    }
-    
-    var splash: UIImage {
-        switch self {
-        case .BlueTheme:
-            return #imageLiteral(resourceName: "StoryboardLaunchScreenProduction")
-        case .TurquoiseTheme:
-            return #imageLiteral(resourceName: "StoryboardLaunchScreenDevelopment")
-        }
-    }
-}
-
-// Enum declaration
-let SelectedThemeKey = "SelectedTheme"
-
-// This will let you use a theme in the app.
-class ThemeManager {
-    
-    // ThemeManager
-    static func currentTheme() -> Theme {
-        if let storedTheme = (UserDefaults.standard.value(forKey: SelectedThemeKey) as AnyObject).integerValue {
-            return Theme(rawValue: storedTheme)!
-        } else {
-            return .TurquoiseTheme
-        }
-    }
-    
-    static func applyTheme(theme: Theme) {
-        // First persist the selected theme using NSUserDefaults.
-        UserDefaults.standard.setValue(theme.rawValue, forKey: SelectedThemeKey)
-        UserDefaults.standard.synchronize()
```

Comment:
FYI if you're ever working on another iOS project: The [docs for `NSUserDefaults.synchronize`](https://developer.apple.com/documentation/foundation/nsuserdefaults/1414005-synchronize?language=objc) say that "this method is unnecessary and shouldn't be used."

Code level comments offer suggestions for migrating to other methods and/or just removing the call altogether. Can't link to the source but someone helpfully copied them [here](https://stackoverflow.com/a/47648278/681493).